src/pyocr/builders.py

```diff
@@ -381,12 +383,23 @@ def __init__(self):
 
         self.__current_box_position = None
         self.__current_box_text = None
+        self.__current_box_confidence = None
         self.boxes = []
 
         self.__current_line_position = None
         self.__current_line_content = []
         self.lines = []
 
+    @staticmethod
+    def __parse_confidence(title):
+        for piece in title.split("; "):
+            piece = piece.strip()
+            if not piece.startswith("x_wconf"):
+                continue
+            confidence = piece.split(" ")[1]
+            return int(confidence)
+        raise Exception("Invalid hocr confidence measure: %s" % title)
```

Comment:
1) This will be a problem. [Paperwork](https://github.com/openpaperwork/paperwork-backend) uses this code to write and read hOCR files. In other words, there are already a lot of people (me included) with a lot of documents/hOCR files written without the confidence.
This code must not raise an exception if the confidence is not found. However it can display a trace instead (this is not really unexpected nor a problem --> not a warning --> `logger.info`).
In other words, please do no break the compatibility (API or file formats) :-)

2) The message is not correct. The problem here is not that the confidence is invalid. The problem is that it hasn't been found.

Reply:
Corrected!